Dockerfile.windows

```diff
@@ -3,14 +3,14 @@
 # Initial stage which pulls prepares build dependencies and CLI tooling we need for our final image
 # Also used as the image in CI jobs so needs all dependencies
 ####################################################################################################
+
+ARG IMAGE_OS_VERSION=1809
+
 # had issues with official golange image for windows so I'm using plain servercore
 FROM mcr.microsoft.com/windows/servercore:ltsc2019 as builder
-ENV GOLANG_VERSION=1.13.4
+ENV GOLANG_VERSION=1.13.15
```

Comment:
I wonder if this should be set to 1.15.7 like we use elsewhere?

Reply:
`release-2.12` where I based the branch on used 1.13.4 also for the normal `Dockerfile`. Wasn't sure whether I should upgrade to a newer major version if the standard one uses the same 1.13 release.. 

Of course that probably means forward porting to the current master might be a little difficult? Not sure how that usually works in this project though :)